### ISSUE TITLE
tabbing through dataset's time period start and end fields marks window as dirty

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#80]
Found a couple of places where using the keyboard causes edit windows to get marked as dirty when no changes have been made and then prompt for confirmation when closing the window. I'm marking these as low priority since they're more quality-of-life improvements rather than issues blocking functionality.

Steps to reproduce:

Select a dataset that has a time period set, like an inventory
Edit the dataset's properties
Click on the Dataset Type field, then hit Tab to move into the Time Period Start field
The window is now incorrectly marked as dirty: window title has an asterisk, clicking Close prompts to discard changes

If the dataset doesn't have a time period set, then tabbing into the Time Period fields doesn't change the dirty status
